plugin/src/main/java/com/denizenscript/denizen/scripts/commands/world/SchematicCommand.java

```diff
@@ -104,7 +104,7 @@ public void run(ReplaceableTagEvent event) {
     // This takes an optional duration as "fake_duration" for how long the fake blocks should remain.
     //
     // The "create" and "paste" options allow the "entities" argument to be specified - when used, entities will be copied or pasted.
-    // At current time, entity types included will be: Paintings, ItemFrames, ArmorStands.
+    // At current time, entity types included will be: Paintings, ItemFrames, GlowItemFrames, ArmorStands, and DisplayEntities.

```

Comment:
Is `DisplayEntities` implemented?

Reply:
Ah good point... Display Entities were already included in the types to be copied so the meta update for it was just to document what already existed. However after adding display entities to the test schematic I'm seeing them getting updated in unexpected ways, losing transformation data, and text_displays losing text.

After disabling the thrown exception on `EntityHelper.setStepHeight` (not all entity mechanisms are applied otherwise) it does actually spawn display entities, however rotation transformation is still completely lost - wondering if this might be a v1.20.6 issue? I'm not able to spawn in display entities with pitch/yaw ever applied either.

Want me to remove the DisplayEntities note for the moment?